docs/ref/models/conditional-expressions.txt

```diff
@@ -209,6 +211,25 @@ What if we want to find out how many clients there are for each
     >>> Client.objects.aggregate(
     ...     regular=Count('pk', filter=Q(account_type=Client.REGULAR)),
     ...     gold=Count('pk', filter=Q(account_type=Client.GOLD)),
-    ...     platinum=Count('pk', filter=Q(account_type=Client.PLATINUM)
+    ...     platinum=Count('pk', filter=Q(account_type=Client.PLATINUM)),
     ... )
     {'regular': 2, 'gold': 1, 'platinum': 3}
+
+
+On backends that support it this aggregate will produce a query leveraging
+the SQL 2003 FILTER WHERE syntax and produce a query similar to this::
+
+    SELECT count('id') FILTER (WHERE account_type=1) as regular,
```

Comment:
It's a small thing, but the following should be added to the SQL blocks.

`.. code-block:: sql`